src/main/java/com/greghaskins/spectrum/internal/hooks/Hooks.java

```diff
@@ -95,25 +96,34 @@ private Hook createChainOfResponsibility(Variable<Boolean> hooksRememberedToRunT
     return chainOfResponsibility;
   }
 
-  private void executeChain(Description description, RunReporting<Description, Failure> reporting,
-      Block block, Hook chainOfResponsibility)
-      throws Throwable {
+  private void executeChain(final Description description,
+      final RunReporting<Description, Failure> reporting,
+      final Block block, final Hook chainOfResponsibility) throws Throwable {
     chainOfResponsibility.accept(description, reporting, block);
   }
 
-  private Hook innerHook(Variable<Boolean> hooksRememberedToRunTheInner) {
-    return (description, notifier, innerBlock) -> {
+  private Hook innerHook(final Variable<Boolean> hooksRememberedToRunTheInner) {
+    return withUnreportedInnerBlock((description, reporting, block) -> {
       hooksRememberedToRunTheInner.set(true);
-      innerBlock.run();
-    };
+      block.run();
+    });
   }
 
   private Hook wrap(final Hook inner, final HookContext outer) {
     return (description, reporting, block) -> outer.getHook().accept(description, reporting,
-        wrapWithReporting(description, reporting,
+        conditionallyWrapWithReporting(inner, description, reporting,
             () -> inner.accept(description, reporting, block)));
   }
 
+  private static Block conditionallyWrapWithReporting(final Hook forHook, final Description description,
+      final RunReporting<Description, Failure> reporting, final Block innerBlock) {
+    if (forHook.requiresUnreportedInnerBlock()) {
+      return innerBlock;
+    }
```

Comment:
This is kind of a workaround and kind of an efficiency - some hooks can ask for no wrapping with additional notification/error handling.